src/main/scala/com/github/takezoe/slick/blocking/BlockingProfile.scala

```diff
@@ -56,7 +57,7 @@ trait BlockingJdbcProfile extends BlockingRelationalProfile { profile: JdbcProfi
       }
     }
   
-    implicit class RepQueryExecutor[E, U, R, T](rep: Rep[E])(implicit unpack: Shape[_ <: FlatShapeLevel, Rep[E], U, R]){
```

Comment:
this change was to make the code compile in scala 2.10, however it is a better signature for this implicit class